src/main/java/io/confluent/connect/jdbc/validation/JdbcSourceConnectorValidation.java

```diff
@@ -442,4 +516,30 @@ protected void addConfigError(String configName, String errorMessage) {
         .ifPresent(cv -> cv.addErrorMessage(errorMessage));
   }
 
-}
+  /** Validate that provided query strings start with a SELECT statement. */
+  private boolean validateSelectStatement(String statement, String configKey) {
+    String trimmedStatement = statement.trim();
+    if (!SELECT_STATEMENT_PATTERN.matcher(trimmedStatement).find()) {
+      String msg =
+          String.format(
+              "Only SELECT statements are supported for '%s'. Please provide "
```

Comment:
This will print the message on UI with `query.masked`. However, the exposed config is `query` both on CC and CP. Better to just use `query` everywhere being agnostic of the specific type of query config that is being used.

Reply:
Yes, on cloud it's unlikely. But still let's make sure that all validation error messages use "query" and not "query.masked"